packages/types/src/index.ts

```diff
@@ -1,4 +1,5 @@
-import * as React from "react";
+import type { ComponentType, ComponentPropsWithoutRef } from "react";
+export * from "./component-types";
```

Comment:
This looks weird

Reply:
In what sense? It says "reexport everything from this file".  You can manually de-structure them all out if you think its clearer, I just didn't have any reason to do it.

That file is separate because the parser script expects it to look a certain way.